samples/wifi/shell/README.rst

```diff
@@ -94,6 +94,8 @@ Supported CLI commands
        | No argument - Prints current configuration
        | on - Turns on power save feature
        | off - Turns off power save feature
+   * - ps_mode
+     - Configure Wi-Fi power save mode legacy/wmm
```

Comment:
Can't we list the values for legacy and wmm? Also wmm should be WMM.
e.g.
- | Configure Wi-Fi power save mode
  | 0 - Legacy
  | 1 - WMM